trappy/base.py

```diff
@@ -179,6 +179,12 @@ def append_data(self, time, comm, pid, cpu, line, data):
         self.line_array.append(line)
         self.data_array.append(data)
 
+    def conv_to_int(self, value):
```

Comment:
I think we can make the handling of other types as a future addition if you don't mind. Also about the naming, I will change it to `cast_to_int()`. Hope that sounds less ugly.